update public docsite to include info about the react-icons-mdl2 package (#16354)
microsoft/fluentui: apps/public-docsite/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

You can use Fluent UI's icons in a few ways, depending on if you're using Fluent UI React or Fabric Core.

> ### Fluent UI React: Font-based Icons

Reviewer: This should use title case for "icons" (I forget whether "font" should also be lowercase or not...)
```suggestion
### Fluent UI React: Font-based icons
```

ReactDOM.render(<MyIconButton />, document.body.firstChild);
```

> ### New SVG Based Fluent UI Icons

Reviewer: You should probably change the "Fluent UI React" heading (earlier) to say "Fluent UI React: font-based icons" and this one to say "Fluent UI React: SVG-based icons". Either that or add level 4 headings "Font-based icons" and "SVG-based icons" under the level 3 "Fluent UI React" heading. (Also, headings are sentence case not title case.)

> #### Fluent UI React: SVG-based Icons

Reviewer: ```suggestion
#### Fluent UI React: SVG-based icons
```

Fluent UI recently released an SVG-based icon set which allows you to import and bundle only the icons you need. This results in smaller bundle sizes compared to the font-based approach with `initializeIcons`, which downloads all icons by default.

> Each SVG icon is wrapped with a React Element and can be imported and used as follows:

Reviewer: ```suggestion
Each SVG icon is wrapped with a React element and can be imported and used as follows:
```

> There is a new Fluent UI Icon set that is an svg based icon solution. This is an iteration on the font based Fluent UI Icon set in which you need to call `initializeIcons` to have access to the icons you want to use in your program.
> 
> The idea of the new icon library is that they are svg icons wrapped within a react element, so you can import the exact icons you need, and not have to rely on calling `initializeIcons`, which will decrease overall bundle size

Reviewer: ```suggestion
Fluent UI recently released an SVG-based icon set which allows you to import and bundle only the icons you need. This results in smaller bundle sizes compared to the font-based approach with `initializeIcons`, which bundles all icons by default.

Each SVG icon is wrapped in a React element and can be imported and used as follows.
```

```ts
import * as React from 'react';
import * as ReactDOM from 'react-dom';
import { ChevronIcon } from '@fluentui/react-icons-mdl2';

ReactDOM.render(<ChevronIcon />, document.body.firstChild);
```

> 

Reviewer: Under "Fluent UI Icons tool" later, you should probably add a note that the tool is only for use with font-based icons currently.

### Fluent UI Icons tool

> The [Fluent UI Icons tool](https://aka.ms/fluentui-icons) lets you search and browse all of Fluent UI's icons. You can also use it to create and maintain subsets of the icon font to use in your web apps, which are drop-in replacements for the default Fabric Core and Fluent UI React icon sets. In addition, the Fluent UI Icons tool is updated with new icons several times a month, whereas the default Fluent UI set is updated only occasionally. You can see detailed docs for the tool at https://aka.ms/fluentui-icons?help. (Note: This tool is only for use with font-based icons currently).

Reviewer: ```suggestion
The [Fluent UI Icons tool](https://aka.ms/fluentui-icons) lets you search and browse all of Fluent UI's icons. You can also use it to create and maintain subsets of the icon font to use in your web apps, which are drop-in replacements for the default Fabric Core and Fluent UI React icon sets. In addition, the Fluent UI Icons tool is updated with new icons several times a month, whereas the default Fluent UI set is updated only occasionally. You can see detailed docs for the tool at https://aka.ms/fluentui-icons?help. (Note: This tool is only for use with font-based icons currently.)
```
